packages/sit-onyx/src/components/OnyxSwitch/types.ts

```diff
@@ -0,0 +1,31 @@
+export type OnyxSwitchProps = {
+  /**
+   * Whether the switch should be checked or not.
+   */
+  modelValue?: boolean;
+  /**
+   * The label of the switch.
+   */
+  label?: string;
+  /**
+   * Whether to disable the switch and prevent user interaction.
+   */
+  disabled?: boolean;
+  /**
+   * Whether the switch should be readonly or not.
+   */
+  readonly?: boolean;
```

Comment:
We don't have a concept for the read-only mode for now, so please remove it for now

Reply:
![image](https://github.com/SchwarzIT/onyx/assets/60856270/cca0255c-4e99-4e83-873e-c6a6a0495507)

What is the fourth column for?

Reply:
Sorry for my imprecision, I meant to say: We don't have a implementation concept for the `readonly` mode yet and didn't decide on how and if we implement it. Relevant ticket: [#339](https://github.com/SchwarzIT/onyx/issues/339)